dashboard/app/controllers/teacher_dashboard_controller.rb

```diff
@@ -18,7 +18,7 @@ def show
     view_options(full_width: true, no_padding_container: true)
   end
 
-  def redirect_to_newest_section
+  def redirect_to_newest_section_progress
```

Comment:
Just checking if this was intended to be in this PR... I thought there was a separate PR for it.

Reply:
This is just renaming the method to be a bit clearer, but I'm happy to remove if there's a different ticket. What task did you think this was part of?